paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -3163,4 +3206,26 @@ public enum MaterializedTableRefreshStatus {
         ACTIVATED,
         SUSPENDED
     }
+
+    /** Actions performed during table writing. */
+    public enum WriteAction {
+
+        // Actions during commit.
+        PARTITION_EXPIRE("partition-expire"),
```

Comment:
Maybe we can add a option for all actions?

Reply:
Here is just an enumeration of all action types